### PR TITLE
KYC acceptance report improvements

### PR DESCRIPTION
**Pull request by akozhevnikov-rebilly**

### CODE REVIEW

File: openapi/paths/reports@kyc-acceptance-summary.yaml [hunk 1]
```diff
@@ -56,13 +56,29 @@ get:
...
                       example: 10
+                    manuallyAccepted:
+                      type: integer
+                      description: Amount of documents that have been accepted by user.
+                      example: 30
+                    rejectedAutoAccepted:
+                      type: integer
+                      description: Amount of documents that have been accepted automatically and then rejected by user.
+                      example: 5
+                    acceptAutoRejected:
```
Comment (adamaltman) [inline openapi/paths/reports@kyc-acceptance-summary.yaml:67]:
```suggestion
                    acceptedAutoRejected:
```

File: openapi/paths/reports@kyc-acceptance-summary.yaml [hunk 2]
```diff
@@ -56,13 +56,29 @@ get:
...
+                    totalAccepted:
+                      type: integer
+                      description: Total accepted documents amount.
+                      example: 100
                     total:
                       type: integer
                       description: Total documents amount.
-                      example: 100
+                      example: 200
                     autoAcceptedPercentage:
```
Comment (adamaltman) [inline openapi/paths/reports@kyc-acceptance-summary.yaml:79]: This needs to be adjusted, and we need to have more percentages to communicate ratios:
```
accuracyRate:
  description: Percent of automatically accepted and rejected documents that remain in the same status after review.
acceptanceRate:
  description: Percent of accepted documents out of the total number of KYC documents.
```

**Review by adamaltman — CHANGES_REQUESTED** [review]

File: openapi/paths/reports@kyc-acceptance-summary.yaml [hunk 3]
```diff
@@ -56,13 +56,33 @@ get:
...
+                      example: 100
                     total:
                       type: integer
                       description: Total documents amount.
-                      example: 100
-                    autoAcceptedPercentage:
+                      example: 200
+                    accuracyRate:
+                      type: number
+                      description: Percent of automatically accepted and rejected documents that remain in the same status after review.
```
Comment (adamaltman) [inline openapi/paths/reports@kyc-acceptance-summary.yaml:81]: From the numbers above, accuracy rate would be:

= (total - acceptedAutoRejected - rejectedAutoAccepted)/total

With some sample numbers:

1000 total
200 accepted that were auto rejected
20 rejected that were auto accepted
600 total accepted

accuracy rate = (1000 - 200 - 20)/1000 = 780/1000 = 78%
acceptance rate = 600/1000 = 60%

Comment (adamaltman) [reply openapi/paths/reports@kyc-acceptance-summary.yaml:81]: (This information may be good in the API description.)
```suggestion
                      description: |
                        Percent of automatically accepted and rejected documents that remain in the same status after review.
                        Calculated by the formula: `(total - accepted auto rejected - rejected auto accepted)/total`.
```

File: openapi/paths/reports@kyc-acceptance-summary.yaml [hunk 4]
```diff
@@ -56,13 +56,33 @@ get:
...
+                      example: 10.00
+                    acceptanceRate:
                       type: number
-                      description: Percents of auto-accepted documents on total amount KYC documents.
+                      description: Percent of accepted documents out of the total number of KYC documents.
```
Comment (adamaltman) [inline openapi/paths/reports@kyc-acceptance-summary.yaml:85]:
```suggestion
                      description: Percent of accepted documents out of the total number of KYC documents. Calculated by accepted total divided by the total.
```

**Review by adamaltman — DISMISSED** [review]

**Review by ashkarpetin — DISMISSED** [review]

**Review by ashkarpetin — DISMISSED** [review]

**Review by mikerebilly — DISMISSED** [review]

**Review by vykhrystiuk — DISMISSED** [review]

File: openapi/paths/reports@kyc-acceptance-summary.yaml [hunk 5]
```diff
@@ -49,21 +49,76 @@ get:
...
+                              type: integer
+                              description: Total amount of accepted documents.
+                              example: 6
+                            automatically:
+                              type: integer
+                              description: Amount of documents that have been accepted automatically.
+                              example: 4
+                            manually:
+                              type: integer
+                              description: Amount of documents that have been accepted by user.
```
Comment (adamaltman) [inline openapi/paths/reports@kyc-acceptance-summary.yaml:74]:
```suggestion
                              description: Amount of documents that have been accepted by reviewer.
```

Comment (adamaltman) [inline openapi/paths/reports@kyc-acceptance-summary.yaml]:
```suggestion
                              description: Amount of documents that have been accepted by reviewer.
```

File: openapi/paths/reports@kyc-acceptance-summary.yaml [hunk 6]
```diff
@@ -49,21 +49,76 @@ get:
...
+                              example: 2
+                            afterAutoRejected:
+                              type: integer
+                              description: Amount of documents that have been rejected automatically and then accepted by user.
```
Comment (adamaltman) [inline openapi/paths/reports@kyc-acceptance-summary.yaml:78]:
```suggestion
                              description: Amount of documents that have been rejected automatically and then accepted by reviewer.
```

Comment (adamaltman) [inline openapi/paths/reports@kyc-acceptance-summary.yaml:78]:
```suggestion
                              description: Amount of documents that have been rejected automatically and then accepted by reviewer.
```
```suggestion
                              description: Amount of documents that have been rejected automatically and then accepted by user.
```

File: openapi/paths/reports@kyc-acceptance-summary.yaml [hunk 7]
```diff
@@ -49,21 +49,76 @@ get:
...
+                              type: integer
+                              description: Total amount of rejected documents.
+                              example: 2
+                            automatically:
+                              type: integer
+                              description: Amount of documents that have been rejected automatically.
+                              example: 1
+                            manually:
+                              type: integer
+                              description: Amount of documents that have been rejected by user.
```
Comment (adamaltman) [inline openapi/paths/reports@kyc-acceptance-summary.yaml:94]:
```suggestion
                              description: Amount of documents that have been rejected by reviewer.
```

Comment (adamaltman) [inline openapi/paths/reports@kyc-acceptance-summary.yaml:94]:
```suggestion
                              description: Amount of documents that have been rejected by reviewer.
```
```suggestion
                              description: Amount of documents that have been rejected by user.
```

File: openapi/paths/reports@kyc-acceptance-summary.yaml [hunk 8]
```diff
@@ -49,21 +49,76 @@ get:
...
+                              example: 1
+                            afterAutoAccepted:
+                              type: integer
+                              description: Amount of documents that have been accepted automatically and then rejected by user.
```
Comment (adamaltman) [inline openapi/paths/reports@kyc-acceptance-summary.yaml:98]:
```suggestion
                              description: Amount of documents that have been accepted automatically and then rejected by reviewer.
```

Comment (adamaltman) [inline openapi/paths/reports@kyc-acceptance-summary.yaml:98]:
```suggestion
                              description: Amount of documents that have been accepted automatically and then rejected by reviewer.
```

**Review by adamaltman — DISMISSED** [review]
Minor text changes to add prior to merge

**Review by ashkarpetin — APPROVED** [review]

**Review by vykhrystiuk — APPROVED** [review]

**Review by adamaltman — APPROVED** [review]

**Pull request merged**